TranslatorReasonerAPI.yaml

```diff
@@ -579,3 +594,70 @@ components:
         URI.
       externalDocs:
         url: https://www.w3.org/TR/2010/NOTE-curie-20101216/
+    MetaKnowledgeGraph:
+      type: object
+      description: >-
+        Knowledge-map representation of this TRAPI web service.
+      properties:
+        nodes:
+          type: object
+          description: >-
+            Collection of node categories provided by this TRAPI web service,
+            indexed by Biolink class CURIEs.
+          additionalProperties:
+            $ref: '#/components/schemas/KMNode'
+        edges:
+          type: array
+          description: >-
+            List of edges/predicates provided by this TRAPI web service.
+          items:
+            $ref: '#/components/schemas/KMEdge'
+    KMNode:
+      type: object
```

Comment:
These values are objects with exactly one property, which begs the question, should the values just be the id-prefix arrays, directly?

Reply:
I would say no, we will also have attributes here as well